docs/expandable/elements.md

```diff
@@ -0,0 +1,49 @@
+## Import
+
+```js
+
+```
+
+## Visual options
+
+#### Default
```

Comment:
Great stuff, just a small detail... Do these need to skip a level?  ### instead of #### ?  would prob look nicer in the short-nav to the right in the docs.